3-cloud-computing/3.0.1-Cloud Service Ecosystems.md

```diff
@@ -0,0 +1,87 @@
+This typically involves self-managed hardware in racks in a datacenter with limited virtualization of resources. 
+
+## IaaS - Infrastructure as a Service
+Infrastructure-as-a-service, or IaaS, is a step away from on-premises infrastructure. 
+It’s a pay-as-you-go service where a third party provides you with infrastructure services, like storage and
+virtualization, as you need them, via a cloud, through the internet.
+
+As the consumer, you are responsible for the operating system, runtimes, data, and applications; 
+a provider gives you access to, and management of, the network, servers, hardware virtualization, 
+and storage you need.
+
+This eliminates the need to maintain your own datacenter. Instead, you access and control the infrastructure 
+via a command line interface (CLI) or web dashboard.
+
+IaaS gives you flexibility to purchase only the components you need and scale them up or down as needed. 
+There’s low overhead and maintenance costs, making IaaS a very affordable option.
+
+Examples of IaaS are AWS EC2, Google CE, and Azure VM.
+
+## CaaS - Container as a Service
+
+Container-as-a-service is a cloud service that offers container-based virtual computing to users, some refer 
+to CaaS as a subset of IaaS. CaaS service providers offer their clients a full-fledged framework where they 
+can perform deployment and management of applications, clusters, and containers.
+Without CaaS, software development teams need to deploy, manage, and monitor the underlying 
+infrastructure that containers run on. 
+
+The CaaS model was created to let developers create, execute, and manage applications that are container-based. 
+A CaaS model is signified by the service provider offering clusters and containers and services.
+
+CaaS enables development teams to think at the higher order container level instead of 
+dealing with all the lower-level infrastructure management. 
+
+Examples of CaaS are AWS ECS, Azure AKS, and Docker Cloud
+
+## PaaS - Platform as a Service
+
+Platform-as-a-service is another step further from full, on-premise infrastructure management. 
+A provider hosts the hardware and software on its own infrastructure and delivers this platform 
+to the user as an integrated solution, solution stack, or service through an internet connection.
+
+Primarily useful for developers and programmers, PaaS allows the user to develop, 
+run, and manage their own apps without having to build and maintain the 
+infrastructure or platform usually associated with the process. External resources, such as
+databases, and queues are configured and attached to runtime containers by the platform. 
+Developers can use built-in provisioned resources to create their applications.
+
+The consumer writes, builds, and manages their apps, but they do it without the headaches of 
+software updates, hardware maintenance or containerization. The consumer pushes an uncontainerize
```

Comment:
Possible misspelling with "uncontainerize"? Possibly "uncontainerized"?

Reply:
changed to un-containerized